fix: skip CSP nonce in style-src when using unsafe-inline (#11575)
sveltejs/kit: .changeset/ninety-clouds-love.md

---
"@sveltejs/kit": patch
---

> fix: Do not automatically add a nonce to the `style-src` directive in the Content Security Policy when the `style-src` directive already contains `unsafe-inline`.

Reviewer: ```suggestion
fix: only add nonce to `style-src` CSP directive when `unsafe-inline` is not present
```
